### PR TITLE
Topic/imx/sai topology

### PR DESCRIPTION
**Pull request by groncarolo**
This small patch set adds SAI topologies both with wm8960 codec and no-codec
It adds a SAI parameter that for now looks like the ESAI but in a near future it will evolve independently
There is a parallel kernel patch that introduces SAI params

### CODE REVIEW

File: src/include/kernel/tokens.h [hunk 1]
```diff
@@ -106,4 +106,7 @@
 /* ESAI */
 #define SOF_TKN_IMX_ESAI_MCLK_ID		1100
 
+/* SAI */
+#define SOF_TKN_IMX_SAI_MCLK_ID			1400
+
```
Comment (dbaluta) [inline src/include/kernel/tokens.h]: After my comment on Linux PR this should be modified and kept in sync.

Comment (groncarolo) [reply src/include/kernel/tokens.h]: done

**Comment by dbaluta** [comment]
@lgirdwood yes! @groncarolo looks like your branch has some conflicts now with master. I've tried resolving them through the web interface. Lets see how it goes.

If it doesn't work I would kindly ask you to rebase your change on latest master.

**Comment by dbaluta** [comment]
@groncarolo now it doesn't work! Please rebase your PR on latest master and resubmit.

File: src/include/kernel/abi.h [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 /** \brief SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 11
+#define SOF_ABI_MINOR 12
 #define SOF_ABI_PATCH 0
```
Comment (dbaluta) [inline src/include/kernel/abi.h]: @groncarolo please also remove ABI increment here.

Comment (groncarolo) [reply src/include/kernel/abi.h]: done

**Pull request merged**